.github/workflows/pylint.yml

```diff
@@ -0,0 +1,25 @@
+name: Run pylint
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install pylint
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install -r requirements-dev.txt
+    - name: Run pylint on all *.py files
+      run: |
+        find . -type f -name "*.py" | xargs pylint
```

Comment:
Aaah this will fail as soon as the first file fails linting (as I described offline). What you want to use is

```
pylint tests libcloudforensics
```
This will tell pylint to lint the `tests` and `libcloudforensics` modules